diff --git a/darts/tests/test_timeseries.py b/darts/tests/test_timeseries.py
index 10551182..4099c2d3 100644
--- a/darts/tests/test_timeseries.py
+++ b/darts/tests/test_timeseries.py
@@ -1117,7 +1117,7 @@ class TestTimeSeries:
         pd_series = pd.Series(range(10), index=times)
         timeseries = TimeSeries.from_series(pd_series)
         resampled_timeseries = timeseries.resample(
-            freq="1h", offset=pd.Timedelta("30T")
+            freq="1h", loffset="30T"
         )
         assert resampled_timeseries.pd_series().at[pd.Timestamp("20200101233000")] == 0
 
